langchain/prompt.py

```diff
@@ -70,3 +73,17 @@ def template_is_valid(cls, values: Dict) -> Dict:
         except KeyError:
             raise ValueError("Invalid prompt schema.")
         return values
+
+    @classmethod
+    def from_examples(
+        cls: Type[TPrompt],
+        examples: List[str],
+        suffix: str,
+        input_variables: List[str],
+        example_separator: str = "\n",
+        prefix: str = "",
+    ) -> TPrompt:
```

Comment:
i think you can just (1) not type hint `cls` and have the return type as `"Prompt"` (like in quotes) and you should be good